src/components/Mapper/Mapper.jsx

```diff
@@ -174,7 +191,7 @@ export default function Mapper() {
       myCanvas.current.removeEventListener('click', handleClick);
       myCanvas.current.addEventListener('mousemove', handleMouseMove);
     };
-  }, [swarm, radar, tod, incense, surfIncense, rod])
+  }, [swarm, radar, tod, incense, surfIncense, rod, pokemon])
```

Comment:
pokemon what? id or monsno?

Reply:
Changed all of the `pokemon` references to be `pokemonName` and derivatives of that.